### DISCUSSION TITLE
Do I need the /dev/net/tun device if I'm using macvlan through docker?

### DISCUSSION THREAD

**Discussion by sofakng** [qdm12/gluetun#1363] [Q&A] [answered]
I'm using gluetun with Docker and I'm also using macvlan networking so that my container is assigned a dedicated ip address on my lan.

I haven't performed any extensive testing and it appears to be working correctly but I'm wondering if I still need to pass the /dev/net/tun device through to the container?

(I also tried removing the NET_ADMIN capability but it complained so I'm leaving that enabled)

**Comment by bnhf** [comment, marked as answer]
@sofakng 

> I haven't performed any extensive testing and it appears to be working correctly but I'm wondering if I still need to pass the /dev/net/tun device through to the container?

I know a /dev/net/tun:/dev/net/tun binding is shown in the docker-compose.yml example, but it is not required -- so you'll be fine. I've never used it.

**Reply by sofakng** [reply]
Thanks much!
